### PR TITLE
[WIP/TESTING] api: Remove v2 protos from `all_protos`

### PR DESCRIPTION
**Pull request by phlax**
Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: .azure-pipelines/pipelines.yml [hunk 1]
```diff
@@ -73,6 +73,11 @@ stages:
 # Presubmit/default
 - ${{ if eq(variables.pipelineDefault, true) }}:
   - template: stages.yml
+    parameters:
```
Comment (htuch) [inline .azure-pipelines/pipelines.yml:76]: PR LGTM when moved to non-WiP (without this unrelated change which I assume goes away then).

Comment (phlax) [reply .azure-pipelines/pipelines.yml:76]: yep - its failing proto_format (which was kinda expected) - so this was there to bypass prechecks
